ow_plexil/src/plans/FaultHandlingPattern1.plp

```diff
@@ -7,14 +7,19 @@
 
 #include "plan-interface.h"
 
+LibraryAction CheckForFaults (InOut Boolean all_ok);
```

Comment:
Same as comment on pattern 2: remove this, and instead do fault check inside or around InitializeAntennaAndArm.  However, I just got a different idea, possibly much better -- will email.